### ISSUE TITLE
Need Drop-Down with Pre-Defined Ticket States

### ISSUE THREAD

**Issue by heliusblackstar** [elbzero/ticket-train#10]
On the Show and Edit ticket pages, we need a drop down menu with pre-defined ticket states.

Suggested states:
New
Assigned
In Progress
Waiting on Customer Response
Waiting on Vendor
Scheduled/On Hold
Resolved
